docs/examples/base.html

```diff
@@ -380,7 +379,6 @@ <h1 style="text-align: center; color: #28b47f; font-weight: bold">
               }
             }
             larvitar.addDefaultTools();
-            larvitar.setToolActive("Wwwc");
```

Comment:
Come mai è stato tolto?

Reply:
perchè tanto viene settato come active di default in addDefaultTools